app/views/projects/list.scala.html

```diff
@@ -17,68 +16,64 @@
 
 <ul class="list-group project-list">
     @models.map { project =>
-        <li class="list-group-item @project.visibility.cssClass">
-            <div class="project-list-header pull-left">
-                <!-- Avatar -->
-                @utils.userAvatar(
-                    user = project.owner,
-                    call = projectRoutes.showIcon(project.ownerName, project.slug),
-                    clazz = "user-avatar-sm pull-left")
+        <li class="list-group-item project @project.visibility.cssClass">
+            <div class="container-fluid">
+                <div class="row">
+                    <div class="col-xs-12 col-sm-1">
+                    @utils.userAvatar(
+                        user = project.owner,
+                        call = projectRoutes.showIcon(project.ownerName, project.slug),
+                        clazz = "user-avatar-sm")
+                    </div>
+                    <div class="col-xs-12 col-sm-11">
+                        <div class="row">
+                            <div class="col-sm-6">
+                                <a class="title" href="@projectRoutes.show(project.ownerName, project.slug)">@project.name</a>
+                            </div>
+                            <div class="col-sm-6 hidden-xs">
+                                <div class="info minor">
+                                    @defining(project.recommendedVersion) { rv: Version =>
+                                        <span class="stat recommended-version" title="Recommended version">
+                                            <i class="fa fa-diamond"></i>
+                                            <a href="@controllers.project.routes.Versions.show(
+                                                project.ownerName, project.slug, rv.name)">
+                                            @rv.name
+                                            </a>
+                                        </span>
+                                        @rv.dependencies.find(_.pluginId.equals(Dependency.MinecraftId)).map { dep =>
+                                            <span class="stat" title="Minecraft version">
+                                                <strong>MC @dep.version</strong>
```

Comment:
This should probably be removed.

When would Minecraft be a dependency for a mod? When it's using NMS or Mixins? (Relates to #247)

Reply:
I have no idea.

Reply:
we support forge mods in theory

Reply:
Remove this for now and when we get better Forge mod support we can add `Minecraft` tags instead of checking the dependencies. (This is what the Sponge and Forge tags did before I implemented the tag system, but I left this here because I wasn't sure what to do with it.)

Reply:
#510